code/modules/admin/IsBanned.dm

```diff
@@ -39,6 +39,17 @@ world/IsBanned(key,address,computer_id)
 			log_misc("Ban database connection failure. Key [ckey] not checked")
 			return
 
+		//Check if we got a gdpr ban for that person
+		var/DBQuery/bangdprquery = dbcon.NewQuery("SELECT id, created_at FROM ss13_ban_gdpr WHERE ckey_hashed = SHA2(:ckey:,256) OR address_hashed = SHA2(:address:,256) OR computerid_hashed = SHA2(:computerid:,256)")
```

Comment:
A computer ID does not have to be hashed, as it already is a hash and is already a psuedonym as far as I know.